ml-search-server/api/controllers/my-connection.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+    connInfo: {
+        database: 'Test',
+        host: 'marklogicdb5672.cloudapp.net',
+        port: 8000,
+        user: 'sa',
+        password: 'Global@123',
```

Comment:
never commit passwords to git use `<password>` as a placeholder, anyone getting latest will place his/her password.